packages/next/src/build/index.ts

```diff
@@ -2334,14 +2333,15 @@ export default async function build(
                   ]
                 : []),
               BUILD_ID_FILE,
-              turboNextBuild ? IS_TURBOPACK_BUILD_FILE : null,
+              SERVER_FILES_MANIFEST,
               path.join(SERVER_DIRECTORY, NEXT_FONT_MANIFEST + '.js'),
               path.join(SERVER_DIRECTORY, NEXT_FONT_MANIFEST + '.json'),
               ...instrumentationHookEntryFiles,
             ]
               .filter(nonNullable)
               .map((file) => path.join(config.distDir, file)),
             ignore: [] as string[],
+            turbopack: turboNextBuild,
```

Comment:
This should go on the config above similar to `isExperimentalCompile` https://github.com/vercel/next.js/blob/7a7272877c8ff2bdd4cf0937ad2f4ba5e789c0d6/packages/next/src/build/index.ts#L2281

Which gets read here https://github.com/vercel/next.js/blob/7a7272877c8ff2bdd4cf0937ad2f4ba5e789c0d6/packages/next/src/server/base-server.ts#L600

Reply:
Does this also work with `next start`?  Seems it's only available with `output: 'standalone'` but we need this error especially during `next start`.

Reply:
it works with `next start` as well we just need to update here for that https://github.com/vercel/next.js/blob/5dd95e3352482a7f14eb7b006dd0663413901337/packages/next/src/server/next.ts#L240-L242